### PR TITLE
add public page names

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[speedup Q2] Public page: Names (3)](https://trello.com/c/QpROFgBR/7286-speedup-q2-public-page-names-3)

**Comment by DanielPalafox** [comment]
[[speedup Q2] Public page: Biography(2)](https://trello.com/c/PSoI76en/7287-speedup-q2-public-page-biography2)

File: src/app/record/components/top-bar/top-bar.component.html [hunk 1]
```diff
@@ -21,66 +21,188 @@
...
+                  >,</ng-container
+                >
+              </ng-container>
+            </div>
+            <div
+              class="mat-caption other-names-wrapper"
+              *ngIf="expandedContent"
+            >
+              <div class="description" i18n="@@record.otherNames">
+                Other names
```
Comment (leomendoza123) [inline src/app/record/components/top-bar/top-bar.component.html:158]: I think we don't require the code duplications of having two Other names

Comment (leomendoza123) [inline src/app/record/components/top-bar/top-bar.component.html:158]: I think we don't require the code duplications of having two Other names

File: src/app/record/components/top-bar/top-bar.component.ts [hunk 2]
```diff
@@ -56,7 +57,7 @@ export class TopBarComponent implements OnInit, OnDestroy {
     if (!this.isPublicRecord) {
       this.getRecord()
     } else {
-      // TODO SUPPORT PUBLIC VIEW
+      this.getPublicRecord()
```
Comment (leomendoza123) [inline src/app/record/components/top-bar/top-bar.component.ts:60]: I am proposing some improvements here on my last PR https://github.com/ORCID/orcid-angular/pull/759 please review that one before we merge this one.

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

**Pull request merged**